### DISCUSSION TITLE
Emacs lsp-pyright can not complete print function

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/pyright#6004] [General]
As the title said

**Comment by erictraut** [comment]
That's not enough information to diagnose the problem you're seeing. I'm not sure what you mean by "complete print function". 

Perhaps you're talking about the "signature help" that is displayed when you type an open parentheses as part of a call expression? Here's what this looks like in VS Code with pyright installed:

![image](https://github.com/microsoft/pyright/assets/7040122/d53c5388-dd0d-4207-8bcc-f6cc0997f36a)

If my assumption is correct and you're talking about signature help, then it would appear this is an issue with emacs or lsp-pyright, and you should file an issue with those projects.

**Reply by ghost** [reply]
No, not the signature help, just the function name.

**Reply by erictraut** [reply]
Sorry, I don't understand what you mean by "just the function name". Can you provide more details? Are you saying that when you type the first few characters of "print" (say, "pr"), then "print" is not among the completion suggestions? This works fine for pyright when it's installed in VS Code, so I'm guessing that it's a problem with lsp-pyright or the way you've configured it. The maintainers of lsp-pyright may be able to help you diagnose the problem.

![image](https://github.com/microsoft/pyright/assets/7040122/37934f7b-58e6-4d9f-b034-5c0280ecc4c6)

**Reply by ghost** [reply]
Yes, what i mean is what you typed, i will open the issue at `lsp-pyright` project, thanks for your answer.
